CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-### 6.0.4 (Next)
+### 6.0.5 (Next)
 
 * Your contribution here.
 
+### 6.0.4 (Next)
```

Comment:
I think you meant to leave this at 6.0.4.

Reply:
Fixed in a follow-up PR